frontend/src/lang/es.ts

```diff
@@ -391,6 +394,7 @@ export const es: LanguageConfig = {
       },
       tos: {
         header: "Términos de servicio",
+        title: "Condiciones del servicio",
```

Comment:
ref: https://github.com/CDCgov/prime-simplereport/pull/4511/files#r998542926

Reply:
based on laura's feedback can we change the header to "Condiciones del servicio" so that we are consistent? Or I guess just change both the es.ts and en.ts file to use one value?